proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+
+### Formal presentation 
+
+After a proposal is merged, the author will present KPI's and deliverables that define the proposal to the HF Technical track. During the next available Technical Track meeting after acceptance, the author or designated lead (as defined by the proposal) should give a short 5-10min presentation consisting of the following: 
+
+1. **High level summary:** the author will present the high-level abstract for what the proposal intends to accomplish. 
+2. **Timeline:** In what timeline, and should outline possible hurdles and risks that affect the timeline. 
+3. **Resources:** What resources are required in order to take the project from start to completion.
+4.  **Contact:** Who is going to update the HF Technical Track members, and on what schedule.
+
+Once the formal presentation is given, the HF Technical Track members will track the progress over time at our ongoing meetings. This also gives the track members time to introduce themselves and and we can all get to know each other. Currently, HF Technical Track meetings are held weekly on Tuesdays at 11:30am-12:30pm UTC-5 (EST). 
+
+## Structure of the process
+
+The HIP process involves the following parties:
+
+1. The HIP Authors
+2. Relevant stakeholders affected by the change
+3. The HF Technical Track members
+
+### The HIP Authors
+
+Authors are responsible for building consensus within the community and documenting dissenting opinions before the HIP is officially discussed by the HIP HF Technical Track . Their goal is to convince the HF Technical Track that their proposal is useful and addresses pertinent problems in the Haskell ecosystem as well as interactions with already existing features. Authors can change over the life-cycle of the HIP.
+
+### The HF Technical Track
+
+The HF Technical Track is an experienced group of people with knowledge of the Haskell ecosystem, responsible for the strategic technical direction on behalf of the Haskell Foundation. Members are tasked with
+
+  - communicating with the community
+  - weighing in pros and cons of every proposal
+  - accepting, postponing or rejecting the proposal.
+
+HF Technical Track members should be either individuals responsible for a specific part of the Haskell ecosystem, or contributors and committers to parts of it. The members are selected by the HF CTO based on their expertise and implication in the community.
+
+The current HF Technical Track members are:
+
+- Emily Pillmore ([@emilypi](https://github.com/emilypi)), Haskell Foundation
+- Richard Eisenberg ([@goldfirere](https://github.com/goldfirere)), Tweag
+- Michael Snoyman ([@snoyberg](https://github.com/snoyberg)), FPComplete
+- Andrew Lelechenko ([@Bodigrim](https://github.com/Bodigrim)), Barclays
+- Davean Scies ([@davean](https://github.com/davean)), XKCD
+- Edward Kmett ([@ekmett](https://github.com/ekmett)), MIRI
+- Theophile Choutri ([@Kleidukos](https://github.com/Kleidukos)), Scrive
+- Gil Mizrahi (@soupi)
+
+### Voting
```

Comment:
If you, @emilypi, unilaterally select the HFTT, then having these rules around voting seem unenforceable: you could always kick someone off the HFTT just during a vote and then reinstate them. So I think it simplifies matters considerably to make the HFTT a labor-producing and advisory body. We can vote, but our vote is a recommendation to you, who makes the decisions.

Reply:
> make the HFTT a labor-producing and advisory body.

I think this idea is in line with what we want as a foundation.